packages/billing/billing-queries.ts

```diff
@@ -101,7 +101,7 @@ export async function getPlanByStripeId(
   return {
     id: results[0].id,
     name: results[0].name,
-    monthlyPrice: results[0].monthly_price,
+    monthlyPrice: parseFloat(results[0].monthly_price as string),
```

Comment:
Perhaps it would be better to cast the return value of the query to a response type rather than casting each monthly_price to a string?

Reply:
I added a function to mitigate this repetitiveness in repetitiveness in https://github.com/cardstack/boxel/pull/1903/commits/5c822f047cf58523138452b841fa518d45c367fb